node_lambnik/src/tiler/package.json

```diff
@@ -23,8 +23,8 @@
     "bin"
   ],
   "scripts": {
-    "deploy": "yarn transpile && claudia update --set-env POSTGRES_DB=${TILEGARDEN_DB},POSTGRES_HOST=${TILEGARDEN_DB_HOST},POSTGRES_PASSWORD=${TILEGARDEN_DB_PASSWORD},POSTGRES_USER=${TILEGARDEN_DB_USER}",
-    "deploy-new": "yarn transpile && claudia create --region us-east-1 --role lambnik-tiler-dev --api-module bin/api --name tilegarden-${USER} --set-env POSTGRES_DB=${TILEGARDEN_DB},POSTGRES_HOST=${TILEGARDEN_DB_HOST},POSTGRES_PASSWORD=${TILEGARDEN_DB_PASSWORD},POSTGRES_USER=${TILEGARDEN_DB_USER}",
+    "deploy": "yarn transpile && claudia update --profile ${AWS_PROFILE} --set-env POSTGRES_DB=${TILEGARDEN_DB},POSTGRES_HOST=${TILEGARDEN_DB_HOST},POSTGRES_PASSWORD=${TILEGARDEN_DB_PASSWORD},POSTGRES_USER=${TILEGARDEN_DB_USER}",
```

Comment:
I think `--profile ${AWS_PROFILE}` reproduces the default behavior.  It has been working for me to run `claudia` commands with the variable set but without this argument.  Despite the wisdom of ["Explicit is better than implicit"](https://www.python.org/dev/peps/pep-0020/), I'd lean toward just letting the default behavior happen here.